CreatePackagesQuickStart.md

```diff
@@ -30,7 +30,7 @@ Here's a TL;DR quick start version of the package creating tutorial. Follow thes
 * **Test the package**
    * **Testing should probably be done on a Virtual Machine**
    * In your package directory, use:
-      * `choco install package-name -s '$pwd' -f` - powershell
+      * `choco install package-name -s "$pwd" -f` - powershell
```

Comment:
@AnthonyMastrean if it doesn't expand the value, we are in trouble, because choco isn't PowerShell and won't know what `$pwd` is.